src/components/FlightInfoCard/FlightInfoCard.js

```diff
@@ -0,0 +1,125 @@
+import styled from 'styled-components';
+import { CgArrowLongRight } from 'react-icons/cg';
+
+const FlightInfoCard = ({ flightInfo }) => {
+  const {
+    filghtLogo,
+    flightCompany,
+    flightNumber,
+    departureTime,
+    landingTime,
+    departureCity,
+    flightTime,
+    landingCity,
+    isDiscount,
+    personnel,
+    price,
+    baggage,
+  } = flightInfo;
+  return (
+    <FlightInfoCardWrapper>
+      <FlightLogo justify="flex-start">
```

Comment:
```suggestion
      <FlightLogo>
```
굳이 props로 넘길 필요 없다면 스타일드 컴포넌트 내부로 옮기면 어떨까요?

Reply:
네! theme객체 내의 flex가 props를 받게 설계를 했고 FlightLogo내부에서 flex를 가져와 사용하게 했습니다. 그래서 props로 작성을 하였는데 그냥 스타일드 컴포넌트 내로 넣는게 더 깔끔할 것 같습니다.